specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+      "properties": {
+        "actionGroup": {
+          "type": "array",
+          "description": "azns notification group reference.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "emailSubject": {
+          "type": "string",
+          "description": "Custom subject for Azns email"
+        },
+        "customWebhookPayload": {
+          "type": "string",
+          "description": "Custom webhook payload to be send to azns action group"
+        }
+      },
+      "description": "azns notification group"
+    },
+    "MetricTrigger": {
+      "type": "object",
+      "properties": {
+        "thresholdOperator": {
+          "$ref": "#/definitions/ConditionalOperator",
+          "description": "Evaluation operation for Metric -'GreaterThan' or 'LessThan' or 'Equal'."
+        },
+        "threshold": {
+          "format": "double",
+          "type": "number"
+        },
+        "metricTriggerType": {
+          "$ref": "#/definitions/MetricTriggerType",
+          "description": "Metric Trigger Type - 'Consecutive' or 'Total'"
+        },
+        "metricColumn": {
+          "type": "string",
+          "description": "Evaluation of metric on a particular column"
+        }
+      }
+    },
+    "ConditionalOperator": {
+      "type": "string",
+      "enum": [
+        "GreaterThan",
+        "LessThan",
+        "Equal"
+      ],
+      "x-ms-enum": {
+        "name": "ConditionalOperator",
+        "modelAsString": true
+      }
+    },
+    "MetricTriggerType": {
+      "type": "string",
+      "enum": [
+        "Consecutive",
+        "Total"
+      ],
+      "x-ms-enum": {
+        "name": "metricTriggerType",
+        "modelAsString": true
+      }
+    },
+    "AlertSeverity": {
+      "type": "string",
+      "enum": [
+        "1",
+        "2",
+        "3",
+        "4"
+      ],
+      "x-ms-enum": {
+        "name": "AlertSeverity",
+        "modelAsString": true
+      }
+    },
+    "AlertStatus": {
+      "type": "string",
+      "enum": [
+        "Active",
+        "Inactive"
+      ],
+      "x-ms-enum": {
+        "name": "AlertStatus",
+        "modelAsString": true
+      }
+    },
+    "QueryType": {
+      "type": "string",
+      "enum": [
+        "ResultCount",
+        "Number"
+      ],
+      "x-ms-enum": {
+        "name": "QueryType",
+        "modelAsString": true
+      }
+    },
+    "LogSearchRule": {
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of the Log Search rule."
+        },
+        "Enabled": {
+          "type": "string",
+          "description": "The flag which indicates whether the Log Search rule is enabled. Value should be true or false",
+          "enum": [
+            "true",
+            "false"
+          ],
+          "x-ms-enum": {
+            "name": "Enabled",
+            "modelAsString": true
+          }
+        },
+        "skuType": {
```

Comment:
SKUs of a resource is indicated through the ARM top level "sku" property. The details can be found in the ARM resource provider contract